.github/workflows/release.yml

```diff
@@ -150,6 +152,92 @@ jobs:
+            const prerelease = parts[2] ? !isNumber(parts[2]) : false;
+            return { prefix, feature, maintenance, build, isOSS, isEE, prerelease };
+          }
+
+          const byVersions = (tag1, tag2) => {
+            const v1 = parseVersion(tag1);
+            const v2 = parseVersion(tag2);
+            const order1 = v1.feature * 1e6 + v1.maintenance * 1e3 + (v1.build || 0);
+            const order2 = v2.feature * 1e6 + v2.maintenance * 1e3 + (v2.build || 0);
+            return order2 - order1;
+          }
+
+          const currentVersion = parseVersion(currentTag);
+          console.log("Current version for", currentTag, "is", currentVersion);
+          if (currentVersion.prerelease) {
+            console.log("This is a pre-release!");
+            console.log("There is no need to tag the container image as latest.");
+            return "not-latest";
+          }
+
+          console.log();
+          console.log("Enumerating all git tag...");
+          const allTags = execSync("git tag -l").toString("utf-8").split("\n");
+          console.log("Found total", allTags.length, "tags");
+
+          console.log();
+          console.log("Filtering for", currentVersion.isOSS ? "OSS" : "EE");
+          const relevantTags = allTags.filter(tag => {
+            const { prefix } = parseVersion(tag);
+            return prefix === currentVersion.prefix;
+          });
+          console.log("Found total", relevantTags.length, "filtered tags");
+          if (relevantTags.length < 10) {
+            console.error("Something is NOT RIGHT! Please check all git tags!!!");
+            return "not-latest";
+          }
+
+          console.log();
+          console.log("Sorting tags to find the highest version numbers...");
+          const sortedTags = relevantTags.sort(byVersions)
+          console.log("Showing 20 tags with the highest versions...");
+          const topTags = sortedTags.slice(0, 20);
+          topTags.map(tag => {
+            const marker = tag === currentTag ? "--->" : "    ";
+            console.log(marker, tag);
+          });
+
+          const latestTag = topTags[0];
+          const isLatest = latestTag === currentTag;
+          console.log();
+          if (isLatest) {
+            console.log("Thus, the container image for ", currentTag, "must be marked as latest.");
+          } else {
+            console.log("The latest container image stays as", latestTag);
+            console.log("There is no need to tag the", currentTag, "container image as latest.");
+          }
+          return isLatest ? "latest" : "not-latest";
+
+    - name: Tag the container image as latest
+      if: ${{ steps.latest_version_check.outputs.result == 'latest' }}
+      run: |
+        echo "Pushing ${{ env.DOCKERHUB_REPO }}:latest ..."
+        docker tag localhost:5000/local-metabase:${{ github.ref_name }} ${{ env.DOCKERHUB_REPO }}:latest
+        docker push ${{ env.DOCKERHUB_REPO }}:latest
+        echo "Finished!"
```

Comment:
I was thinking to maybe omit this part for the upcoming release and to just observe the logs instead. But thinking some more about it, it doesn't really matter. Even if something goes wrong, we can always override the `latest` docker tag manually.